README.md

```diff
@@ -83,6 +84,29 @@ If you have multiple JFrog platform configurations as secrets, you can use all o
 | Important: When exposing more than one JFrog configuration to the Action, you should always add the ```jf c use``` command to specify the server to use. |
 | --- |
 
+### Setting direct connection details environment variables
+You can use connection details directly by settings one of the following environment variables combinations:
```

Comment:
You can use connection details directly by settings one of the following environment variables combinations:
-->
You can set the connection details to your JFrog Platform by using one of the following environment variables combinations: